interactions/api/http/emoji.py

```diff
@@ -32,6 +32,16 @@ async def get_guild_emoji(self, guild_id: int, emoji_id: int) -> dict:
         """
         return await self._req.request(Route("GET", f"/guilds/{guild_id}/emojis/{emoji_id}"))
 
+    async def get_emoji(self, guild_id: int, emoji_id: int) -> dict:
+        """
+        Gets an emote from a guild.
```

Comment:
It should be mentioned that this is an alias to `get_guild_emoji`, like how [`get_self`](https://github.com/interactions-py/library/blob/1d3ec28aa5d8bd1b8cc6c52c15960e118b6a8cc8/interactions/api/http/user.py#L20) does.

Though, honestly, I do question why `get_self` exists in the first place. It's weird and unnecessary. This alias *may* be, too.

Reply:
This was mainly for consistency and for the future `get()` method since roles and emojis were the only outliers.
I will mention that it is an alias.